### PR TITLE
Feature/circular_stats

### PR DESCRIPTION
**Pull request by k-fleck-01**
Added circular statistics (mean, variance and standard deviation) implementation to "src/output/stats.rs". Added an updated example, "stats.yml", that includes both circular statistics and the expressions subkey - would replace the "stats_expr.yml" from the stats-expr PR ideally.

Checked that circular results agree with linear statistics in the small angle approximation too.

### CODE REVIEW

**Comment by tgblackburn** [comment]
Thanks Kyle, looks good. Again, a brief explanation of the new stats functions in input.md would be great.

**Comment by k-fleck-01** [comment]
Added the extra choices "circvariance" and "cvariance" for the circular reduction and added some info to the input.md about the new circular stats options.

**Review by tgblackburn — APPROVED** [review]

File: src/output/stats.rs
[the remark this answers is not in the document]
Comment (k-fleck-01) [reply src/output/stats.rs:1]: I can try implementing an angle check, not sure the best way to go about that. And I did not. know the f64::hypot() method - rather nifty!
And good catch with the units, I missed that.

[the remark this answers is not in the document]
Comment (tgblackburn) [reply src/output/stats.rs:1]: No worries, I will add this post-merge.

**Pull request merged**